src/main/java/frc/robot/commands/IntakeDeploy.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeDeploy extends CommandBase {
+  private final IntakeSubsystem m_intakesubsystem;
+  /** Creates a new IntakeDeploy. */
+  public IntakeDeploy(IntakeSubsystem intakesubsystem) {
+    addRequirements(intakesubsystem);
+    m_intakesubsystem = intakesubsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    m_intakesubsystem.deployIntake();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return true;
```

Comment:
This will run only once with isFinished set to true..

Is this what we want to happen?  One press equals a swap in state of the intake?
We could also consider whether we retract the intake while a button is held.